sys-apps/irqbalance/files/irqbalance.service

```diff
@@ -2,10 +2,8 @@
 Description=CPU Interrupt Balancer
 
 [Service]
-ExecStart=/usr/sbin/irqbalance --debug
-# --debug implies a lot of cruft
-# but it's the only way to get foregrounding
-StandardOutput=null
+EnvironmentFile=-/etc/irqbalance.env
+ExecStart=/usr/sbin/irqbalance --foreground $IRQBALANCE_ARGS
```

Comment:
Please double-check that `--foreground` works correctly. I'm pretty sure I had some reason to use `--debug` instead.